src/domain/install/controllers/class.index.php

```diff
@@ -0,0 +1,100 @@
+<?php
+
+namespace leantime\domain\controllers {
+
+    use leantime\core;
+    use leantime\domain\repositories;
+    use leantime\domain\services;
+    use leantime\domain\models;
+
+    class index
+    {
+
+        private $tpl;
+        private $usersService;
+        private $redirectUrl;
+
+
+        /**
+         * constructor - initialize private variables
+         *
+         * @access public
+         * @params parameters or body of the request
+         */
+        public function __construct()
+        {
+
+            $this->tpl = new core\template();
+            $this->installRepo = new repositories\install();
+
+            if ($this->installRepo->checkIfInstalled()) {
+               core\frontcontroller::redirect(BASE_URL);
+            }
+
+        }
+
+
+        /**
+         * get - handle get requests
+         *
+         * @access public
+         * @params parameters or body of the request
+         */
+        public function get($params)
+        {
+
+            $this->tpl->display("install.new", 200, "entry");
+
+        }
+
+        public function post($params) {
+
+            $values = array(
+                'email'			=>"",
+                'password'		=>"",
+                'firstname'		=>"",
+                'lastname'		=>""
+            );
+
+            if(isset($_POST['install'])) {
+
+                $values = array(
+                    'email' => ($params['email']),
+                    'password' => $params['password'],
+                    'firstname' => ($params['firstname']),
+                    'lastname' => ($params['lastname']),
+                    'company' => ($params['company'])
+                );
+
+                if (isset($params['email']) == false || $params['email'] == '') {
+                    $this->tpl->setNotification("notification.enter_email", "error");
+                } else {
```

Comment:
I believe `if->elseif->elseif` is slightly more efficient and easier to read than a series of nested `else->if->else->if` statements